avogadro/vtk/vtkglwidget.cpp

```diff
@@ -50,7 +50,7 @@ vtkImageData* createCubeImageData(Core::Cube* cube)
 
   data->AllocateScalars(VTK_FLOAT, 1);
 
-  auto* dataPtr = static_cast<double*>(data->GetScalarPointer());
+  auto* dataPtr = static_cast<float*>(data->GetScalarPointer());
```

Comment:
I see commit 826904ae7391b842a167a474f5003587e5c6027c broke this where the committer changed two of the three types, it likely never worked and simply was not tested before committing in July 2022.